### PR TITLE
Improve dash

### PR DESCRIPTION
**Pull request by dandavies99**
Some improvements to the Dash app itself, which should help with development. 
- Mount the dash app directory and use the gunicorn `--reload` flag so that changes to code are reflected in the app.
- Set the background colour of the overall app so the text is visible when viewed via an OVE html app. 
- Add a footer in the bottom right to keep track of the exact limits of the OVE section.

### CODE REVIEW

File: dash/app.py [hunk 1]
```diff
@@ -17,7 +18,8 @@
             ]
         ),
         dash.page_container,
-    ]
+        html.H2("Footer", style={"float": "right", "backgroundColor": "#F9F9F9"}),
```
Comment (AdrianDAlessandro) [inline dash/app.py:21]: This footer is positioned directly beneath the plot for me. Is it meant to extend to the bottom of the available space?
Screenshot 2023-02-03 at 14 58 55

**Review by AdrianDAlessandro — APPROVED** [review]
A couple of small questions, but I think this is worth merging despite them

**Comment by dandavies99** [comment]
OK I've updated `configure.py` to mount the volume to the right place in the generated `docker-compose.yml`. 

The only caveat for the "hot reloading" as far as I can see is that if one of the files in `pages/` is changed, it doesn't trigger an automatic reload. Only saving (even there are no changes) `app.py` itself triggers the changes to be reflected in the app. This is slightly annoying but as far as I can tell it's a limitation of using pages with Dash. 

@AdrianDAlessandro @CWestICL - feel free to see if these changes work for you locally.

**Comment by AdrianDAlessandro** [comment]
I'm merging this, but I Had to manually edit the docker-compose.yml on the server instead of running configure.py because there is no python install on the server. How did you generate it to begin with @dandavies99 ?

**Pull request merged**